### ISSUE TITLE
Cant play any games or start Razer Synapse anymore.

### ISSUE THREAD

**Issue by HansPeter321** [mrpond/BlockTheSpot#189]
### Describe the bug
A clear and concise description of what the bug is, along with your OS [eg. Windows 10]:

After installing Block the Spot. I dont have advertisements on spotify, but I have a new big problem now. Every time  I start a game  or a other program I get a error. It has something to do with my RAM. I had never got this Problem befor. I cant play Rainbow six or Minecraft or use my Razer Synapse.

I have deleted Spotify and Block the Spot, but i still have this problem. And when i reinstall Spotify I still dont have adveertisements, although I haven't reinstalled Block the Spot.

I have Windows 10.
![image](https://user-images.githubusercontent.com/87834269/126713458-fe08f345-84e8-4ce0-ad97-be38df9b2009.png)
Minecraft error:
![image](https://user-images.githubusercontent.com/87834269/126713912-499f1b63-e9bd-4844-93bf-8a314efa7093.png)

**Comment by Celuni** [comment]
Why do you assume that has anything to do with BlockTheSpot?
Corrupted RAM maybe. How often do you restart your PC, is it probably running 24/7 then there is your issue.

**Comment by Nuzair46** [comment]
This is completely unrelated to BlockTheSpot and It is more likely caused by some other issue related to your PC.
Please clarify this by uninstalling BlockTheSpot and just using Spotify to see if the issue persist.

**Comment by HansPeter321** [comment]
> This is completely unrelated to BlockTheSpot and It is more likely caused by some other issue related to your PC.
> ~Please clarify this by uninstalling BlockTheSpot and just using Spotify to see if the issue persist.~
> 
> Since you have uninstalled Spotify and the issue still persists, Its not and issue caused by BlockTheSpot. Contact respective support centre to fix this issue.

I have removed BlockTheSpot but I still dont have advertisements on Spotify. And this problem startet after installing BlockTheSpot. BlockTheSpot has to be still working somehow on my PC.

**Comment by HansPeter321** [comment]
> Why do you assume that has anything to do with BlockTheSpot?
> Corrupted RAM maybe. How often do you restart your PC, is it probably running 24/7 then there is your issue.

Bc the Spotify error is something about RAM and no I dont run my PC 24/7.

**Comment by Nuzair46** [comment]
use uninstall.bat to remove BlockTheSpot.
if done already, reinstall Spotify.
Doing that should definitely remove blockthespot

**Comment by HansPeter321** [comment]
> use uninstall.bat to remove BlockTheSpot.
> if done already, reinstall Spotify.
> Doing that should definitely remove blockthespot

Paji. I did that. I used unistall.bat and the cmd tould me that it was successful. After using unistall.bat and uninstalling Spotify I looked for any leftover Spotify stuff but it was all clean. And after reinstalling Spotify from a fresh setup I still dont have any advertisements. All this Problem startet after using BlockTheSpot but it could also be a coincidence. I dont want to waste your time anymore and will install my win 10 new. 

Have a nice day Bro. (I am also a Indian but was born in Austria and live there.)
